plugin.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"code.cloudfoundry.org/cli/plugin"
+)
+
+var (
+	conn plugin.CliConnection
+)
+
+type Plugin struct {
+	cmd *cobra.Command
+}
+
+func (p *Plugin) Run(c plugin.CliConnection, args []string) {
+	conn = c // FIXME: this isn't great, can we pass into p.cmd somehow?
```

Comment:
Why are we holding a package-global reference to this? If it's unused, can we remove it?

Reply:
It _was_ still being used by the code that fetched the ssh password. I have now implemented fetching the ssh-code via the client, and pass all the required plugin data through as command flags to avoid the ugly `CliConnection` global.